package.json

```diff
@@ -74,7 +74,7 @@
 		"icon": "rimraf assets/generated && electron-icon-maker --input=assets/youtube-music.png --output=assets/generated",
 		"generate:package": "node utils/generate-package-json.js",
 		"postinstall": "yarn run icon && yarn run plugins",
-		"clean": "del dist",
```

Comment:
What's wrong with `del`? https://github.com/th-ch/youtube-music/pull/932

Reply:
I assume del wasn't used because rimraf was used everywhere else i will fix that soon.

Reply:
rimraf has been replaced by del because it created issues on Windows https://github.com/th-ch/youtube-music/pull/932